app/controllers/votes_controller.rb

```diff
@@ -1,38 +1,37 @@
 class VotesController < ApplicationController
   before_action :authenticate_user!
-  before_action :set_target
+  before_action :set_vote
 
   def up
-    @target.increment(:score)
-    if @target.valid?
-      @target.save!
-      render json: @target.votable.sum_votes
+    @vote.increment(:score)
+    if @vote.valid?
+      @vote.save!
+      render json: @vote.votable.sum_votes
     else
       render json: 'you are already voted.', status: :unprocessable_entity
     end
   end
 
   def down
-    @target.decrement(:score)
-    if @target.valid?
-      @target.save!
-      render json: @target.votable.sum_votes
+    @vote.decrement(:score)
+    if @vote.valid?
+      @vote.save!
+      render json: @vote.votable.sum_votes
     else
       render json: 'you are already voted.', status: :unprocessable_entity
     end
-
   end
 
   private
 
-    def set_target
+    def set_vote
```

Comment:
Assignment Branch Condition size for set_vote is too high. [16.52/15]<br>Inconsistent indentation detected.